sensor_msgs/Image datasetImage
float64 timeStamp